api/src/main/java/org/apache/cloudstack/api/response/DirectDownloadCertificateHostStatusResponse.java

```diff
@@ -0,0 +1,73 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package org.apache.cloudstack.api.response;
+
+import com.cloud.serializer.Param;
+import com.google.gson.annotations.SerializedName;
+import org.apache.cloudstack.api.ApiConstants;
+import org.apache.cloudstack.api.BaseResponse;
+
+public class DirectDownloadCertificateHostStatusResponse extends BaseResponse {
```

Comment:
@nvazquez 
this class is quite similar as [DirectDownloadCertificateHostMapResponse.java](https://github.com/apache/cloudstack/pull/6104/files#diff-ab35227d0c94f157e2c9651fa42a7f719ea96f55e5c1c9fae9c51ce4b2e09188)

would be good to combine them into one ?

Reply:
Sure, thanks